.circleci/config.yml

```
@@ -58,6 +58,20 @@ jobs:
           key: kibit-deps-v1-{{checksum "deps.edn"}}
           paths: [.cpcache, ~/.m2, ~/.gitlibs]
 
+   # Eastwood is Yet Another lint tool for Clojure.
+   tool_eastwood:
+     docker:    # See https://circleci.com/docs/2.0/circleci-images/
+       - image: circleci/clojure:tools-deps
+     working_directory: ~/project/tool
+     steps:
+       - checkout: {path: ~/project}
+       - restore_cache: {keys: ['eastwood-deps-v1-{{checksum "deps.edn"}}']}
+       - run: bin/install-clojure
+       - run: clojure -A:eastwood # downloads deps if needed
```

Suggest adding this comment in a description for the step instead.
```suggestion
       - run:
           command: clojure -A:eastwood
           name: Download Clojure dependencies, if needed
```

---

I see what you mean, I can do something like this. Downloading the dependencies is a side effect of the step, but the name of the step would be a good place to note that. Thanks!

---

Addressed in 3c39516 — thanks!